### PR TITLE
Yoga: Use newer liblasso package version in Ubuntu

### PR DESCRIPTION
**Pull request by cityofships**

### CODE REVIEW

**Review by markgoddard — DISMISSED** [review]

**Comment by markgoddard** [comment]
Has merge conflict, might need a rebase to pick up CI fix

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -250,6 +250,14 @@ kolla_build_blocks:
         && grafana-cli plugins install grafana-piechart-panel
   ironic_inspector_header: |
     ADD additions-archive /
+  keystone_base_footer: |
+    {% raw %}
+    {% if base_package_type == 'deb' %}
+    RUN curl -sLO https://github.com/stackhpc/lasso/releases/download/applied%2F2.7.0-2build3/liblasso3_2.7.0-2build3_amd64.deb && \
```
Comment (mnasiadka) [inline etc/kayobe/kolla.yml]: just use curl, curlrc in base image already has those options.

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -250,6 +250,14 @@ kolla_build_blocks:
...
+        sudo dpkg -i liblasso3_2.7.0-2build3_amd64.deb && \
```
Comment (mnasiadka) [inline etc/kayobe/kolla.yml]: apt?

Comment (mnasiadka) [reply etc/kayobe/kolla.yml]: Actually you can apt install from an url?

Comment (cityofships) [reply etc/kayobe/kolla.yml]: > Actually you can apt install from an url?

Possible with DNF but not APT

File: etc/kayobe/kolla.yml (hunk 1 above)
Comment (markgoddard) [reply etc/kayobe/kolla.yml]: No need to polish here - this works

**Review by markgoddard — DISMISSED** [review]

**Comment by markgoddard** [comment]
Merge conflict again :(

**Comment by cityofships** [comment]
Something went sideways in https://github.com/stackhpc/stackhpc-kayobe-config/pull/443 - one part of this change is merged (build overrides) but not updated image tag.

**Comment by markgoddard** [comment]
> Something went sideways in #443 - one part of this change is merged (build overrides) but not updated image tag.

I didn't pull in the tag because I knew you had this PR open. Perhaps I should have done it in the merge instead.

**Review by markgoddard — APPROVED** [review]
Rebased onto neutron patch to avoid merge conflicts.

**Pull request merged**